utils/compatibility/utils.py

```diff
@@ -0,0 +1,111 @@
+import yaml
+import requests
+
+from collections import OrderedDict
+from colorama import Fore, Style
+from packaging.version import Version
+
+
+def printError(message):
```

Comment:
we should standardize on snake casing functions since that is the python standard